reboot_pending/reboot_pending.resource.ps1

```diff
@@ -0,0 +1,12 @@
+ if (Get-ChildItem "HKLM:\Software\Microsoft\Windows\CurrentVersion\Component Based Servicing\RebootPending" -EA Ignore) { return @{ RebootPending = $true } | ConvertTo-Json -Compress }
```

Comment:
Add a comment to the docs on the reboot registry locations

Reply:
Updated.